sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -482,5 +521,22 @@ public virtual Response ActivateMultiSlot(string eventId, CancellationToken canc
                 throw;
             }
         }
+
+        /// <summary> Gets the configuration details for the live model to use </summary>
+        internal Configuration GetClientConfigurationForLiveModel()
+        {
+            result = new Lazy<Models.PersonalizerClientProperties>(() => ClientConfigurationRestClient.Post());
+            Configuration config = new Configuration();
+            // configure the personalizer loop
+            config["appid"] = result.Value.ApplicationID;
```

Comment:
Instead of call `result.Value` half a dozen times, please use a temporary reference.